<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit_calc</class>
 <widget class="QWidget" name="Credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>411</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Credit_calc</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(68, 235, 153);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resource/images/sber.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../misc/images/sber.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sum">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;
	qproperty-alignment: AlignRight;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_time">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;
	qproperty-alignment: AlignRight;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Срок (мес.)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_percent">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;
	qproperty-alignment: AlignRight;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ann">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(255, 255, 255);
	font: 75 11pt &quot;Ubuntu Condensed&quot;;
	font-size: 25px;</string>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_dif">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(255, 255, 255);
	font: 75 11pt &quot;Ubuntu Condensed&quot;;
	font-size: 25px;</string>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Uroob&quot;;
color: rgb(255, 255, 255);
font-size: 32px;</string>
   </property>
   <property name="text">
    <string>CreditCalc_v2.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Переплата по кредиту</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Общая выплата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;
	qproperty-alignment: AlignRight;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;
	qproperty-alignment: AlignRight;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border: 3px solid gray;
	font-size: 20px;
	qproperty-alignment: AlignRight;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 3px solid gray;
	font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Расчитать условия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 3px solid gray;
	font-size: 20px;
	qproperty-alignment: AlignCenter;</string>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
